Dockerfile

```diff
@@ -0,0 +1,44 @@
+# Container image that runs your code
+FROM php:7-cli
+RUN apt-get update \
+    && apt-get --quiet --yes --no-install-recommends install \
+      keychain \
+      unzip \
+    && curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip" \
+    && unzip awscliv2.zip \
+    && ./aws/install
+
+# Acquia Connection Information
+ARG INPUT_ACQUIA_ENVIRONMENT
+ARG INPUT_ACQUIA_PROJECT
+ARG INPUT_ACQUIA_PRIVATE_KEY
+ARG INPUT_ACQUIA_PUBLIC_KEY
+# AWS Connection Information
+ARG INPUT_AWS_S3_BUCKET
+ARG INPUT_AWS_ACCESS_KEY_ID
+ARG INPUT_AWS_SECRET_ACCESS_KEY
+ARG INPUT_AWS_DEFAULT_REGION
+
+# Authorize SSH Host
+RUN mkdir -p /github/home/.ssh && \
+    chmod 0700 /github/home/.ssh && \
+    touch /github/home/.ssh/known_hosts && \
+    ssh-keyscan acquia-sites.com > /github/home/.ssh/known_hosts
+
+RUN  echo "    IdentityFile ~/.ssh/acquia" >> /etc/ssh/ssh_config
+RUN ssh-keygen -A
+
+# Add the keys and set permissions
+RUN echo "$INPUT_ACQUIA_PRIVATE_KEY" > /github/home/.ssh/acquia && \
```

Comment:
I'm not sure where the issue is, but it seems to be something with this line.

```
Warning: Identity file /github/home/.ssh/acquia not accessible: No such file or directory.
Host key verification failed.
```

https://github.com/ChromaticHQ/imprivata-drupal-8/runs/1892447793?check_suite_focus=true#step:3:50

@markdorison any ideas here after your recent ssh key work?

Reply:
Do you need to create the file first?

```
touch /github/home/.ssh/acquia
```

Reply:
Just tried that. Doesn't seem to work. 🙁 

https://github.com/ChromaticHQ/imprivata-drupal-8/runs/1911708903?check_suite_focus=true